Rubberduck.SourceControl/GitProvider.cs

```diff
@@ -185,15 +185,16 @@ public override IRepository InitVBAProject(string directory)
                 var status = repo.RetrieveStatus(new StatusOptions {DetectRenamesInWorkDir = true});
                 foreach (var stat in status.Untracked)
                 {
-                    repo.Stage(stat.FilePath);
+                    // repo.Stage(stat.FilePath); //deprecated from LibGit2Sharp v0.24
```

Comment:
don't comment code out. remove it. That's what source control is for :)

Reply:
done it here and all below 👍